### ISSUE TITLE
Does transfer learning exist for RPMNet?

### ISSUE THREAD

**Issue by kidpaul94** [yewzijian/RPMNet#19]
Hello again,

Can we do transfer learning with RPMNet to quickly add new objects? Currently, pre-trained weights does not work well for my own object: 
![Screenshot from 2021-05-25 22-49-43](https://user-images.githubusercontent.com/79131554/119595436-92824400-bdab-11eb-9438-b9f2376350b2.png)

 This may happen because my source (orange) miss almost 60% of geometry compared to the target (blue) (pretty harsh condition due to sensor noise and occlusion). However, I want to verify whether pre-trained weights is an issue for this network not showing good performance.

**Comment by yewzijian** [comment]
It should be possible to perform transfer learning by using the provided training code. 

I'm not sure if this is the cause, but you might want to note that we originally trained the network using a point cloud which is normalised to fit into a unit sphere. The provided pertained weights almost definitely won't work well on point clouds of other scales.


Zi Jian

**Comment by kidpaul94** [comment]
Thank you for the quick reply! Indeed, I'm pretty sure that I normalized these two pointclouds before putting into the network, but the model didn't show good performance. Could you elaborate more about the process of transfer learning? At a glance, I couldn't find anywhere in train.py where pre-trained model is loaded and resume the training....

**Comment by yewzijian** [comment]
Hi, you can just pass in the path to the checkpoint using the --resume argument. 
You can find the loading of the  oretained model at [L237 of train.py](
https://github.com/yewzijian/RPMNet/blob/74f6531968bc084e44231e397da640432cd42a4b/src/train.py#L237).


Zi Jian

**Comment by kidpaul94** [comment]
Thank you!
